server/controllers/lecture-controller.js

```diff
@@ -371,7 +372,7 @@ const sendEmailsForCancelledLecture = async (lectureId) => {
       emailBody(
         item.name,
         item.surname,
-        item.lectureName,
+        //item.lectureName,
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.